openedxstats/apps/sites/views.py

```diff
@@ -15,12 +15,18 @@
 from django.core.exceptions import ValidationError
 from django.db.models import Count, Sum, Q
 from django.views.decorators.csrf import csrf_exempt
-
 from openedxstats.apps.sites.models import (
     Site, SiteLanguage, SiteGeoZone, Language, GeoZone, SiteSummarySnapshot,
     AccessLogAggregate, OverCount,
 )
 from openedxstats.apps.sites.forms import SiteForm, LanguageForm, GeoZoneForm
+from django.http import JsonResponse
```

Comment:
A Django import should be put with the other Django imports further up.  Python style says to group imports into three sections:  standard library, third-party modules, our own code.  Within each section they should be ordered alphabetically.